### PR TITLE
🌺 INTEGRATE - Course detail and Self paced view 🌸

### PR DESCRIPTION
**Pull request by shylajah15**
# Pull Request Template

Checks before making a PR

- [x] Is version bumped?

@gkmahendran09  

- Integrated course detail view 
- Integrated self-paced view and it's logic
- Free text field integration for roles and gender
- Script to push the code to AWS repo

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -48,6 +48,22 @@ http {
...
+        }
+
+        location /courses {
+            proxy_pass https://lms-staging.academy.who.int;
+        }
+
+        location /static {
+            proxy_pass https://lms-staging.academy.who.int;
+        }
+
```
Comment (gkmahendran09) [inline nginx.conf:66]: Create an issue we need to get this URL from the config

File: chart/values.yaml [hunk 2]
```diff
@@ -13,14 +13,15 @@ route: true
 # custom end point injected by config map. This is likely to changed
 config_map: '{
   "SSO": {
-  "url": "https://sso-labs-staging.apps.prod.lxp.academy.who.int/auth",
+  "url": "https://sso-staging.academy.who.int/auth",
   "realm": "whoa-lxp",
   "clientId": "web-app",
-  "redirectUri": "https://learning-experience-platform-labs-staging.apps.prod.lxp.academy.who.int/onboarding"
+  "redirectUri": "https://dev-course-api-learning-experience-platform-labs-dev.apps.prod.lxp.academy.who.int/onboarding"
```
Comment (gkmahendran09) [inline chart/values.yaml:19]: Revert back to the staging URL

File: chart/values.yaml [hunk 3]
```diff
@@ -13,14 +13,15 @@ route: true
 # custom end point injected by config map. This is likely to changed
 config_map: '{
   "SSO": {
-  "url": "https://sso-labs-staging.apps.prod.lxp.academy.who.int/auth",
+  "url": "https://sso-staging.academy.who.int/auth",
```
Comment (gkmahendran09) [inline chart/values.yaml:16]: Revert back to OpenShift SSO URL

**Review by gkmahendran09 — APPROVED** [review]
Have added few minor comments but overall looks good to me!

File: nginx.conf (hunk 1 above)
Comment (shylajah15) [reply nginx.conf:66]: @gkmahendran09 Created an issue #52

**Pull request merged**